staff/tachi-fernandez/cv/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Curiiculum Vitae</title>
+</head>
+<body>
+  <p>
+    <h1> Curriculum Vitae => Tachi Fernandez</h1>
+      <h2>ºDatos personales:</h2>
```

Comment:
Maybe I would put the title inside each section, it would be more semantic.

Reply:
ready :+1: